### DISCUSSION TITLE
Create Next App `--`

### DISCUSSION THREAD

**Discussion by Pedromdsn** [pnpm/pnpm#4579] [Q&A]
Why had it `--`on command:

![image](https://user-images.githubusercontent.com/57453260/163606786-6d764c8e-93db-449b-ba1b-aa9b8957e163.png)

**Comment by woss** [comment]
the `--` signifies the end of options for the given command, in your case `pnpm`. everything after that is passed as an argument into the process, in your case `next-app`.

Sources are here https://unix.stackexchange.com/questions/11376/what-does-double-dash-mean
